### PR TITLE
docs: add a table with each feature supported by model

### PR DESCRIPTION
**Pull request by smoya**
I believe it could be interesting to have a table with features supported per each model. This gives you quick visual info about what is and what isn't supported by pgai. I created the table based on the info we have in our README.

This is just a suggestion and I'm very happy to discard it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,19 @@ the foundation of most AI applications, pgai makes it easier to leverage your da
...
+
+| **Model**      | **Tokenize** | **Embed** | **Chat Complete** | **Generate** | **Moderate** | **Classify** | **Rerank** |
+|-----------------|:------------:|:---------:|:-----------------:|:------------:|:------------:|:------------:|:----------:|
+| **Ollama**      |      ❌       |     ✔️     |         ✔️         |      ✔️       |      ❌       |      ❌       |     ❌      |
+| **OpenAI**      |      ✔️       |     ✔️     |         ✔️         |      ❌       |      ✔️       |      ❌       |     ❌      |
+| **Anthropic**   |      ❌       |     ❌     |         ❌         |      ✔️       |      ❌       |      ❌       |     ❌      |
+| **Cohere**      |      ✔️       |     ✔️     |         ✔️         |      ❌       |      ❌       |      ✔️       |     ✔️      |
+
+### Legend:
+- ✔️ : Feature is supported
```
Comment (JamesGuthrie) [inline README.md:50]: Could we use the green tick (✅ ) instead, to offset the red?

**Review by JamesGuthrie — APPROVED** [review]
LGTM, one minor remark which you can ignore.

File: README.md (hunk 1 above)
Comment (smoya) [reply README.md:50]: Yep, good idea. Changing!

**Review by JamesGuthrie — APPROVED** [review]

**Comment by smoya** [comment]
@JamesGuthrie Minor but maybe makes sense to keep the design of the tables in https://github.com/timescale/pgai/blob/main/docs/vectorizer-api-reference.md#parameters and use ✔️ and ✖️ instead?

**Comment by smoya** [comment]
I finally opted by a version without ✖️ and instead leave it empty, relying only in ✔️ . It improves readability **by a lot**: 

CleanShot 2024-11-28 at 11 37 02@2x

I additionally updated the adding-embedding-integration.md file so we add a new row into this table on each new integration.

cc @JamesGuthrie as you already reviewed it.

File: docs/adding-embedding-integration.md [hunk 2]
```diff
@@ -52,8 +52,9 @@ testing we would like.
 
 ## Documentation
 
-Ensure that the new integration is documented. Document the new database
-function in [vectorizer-api-reference.md], and changes to the vectorizer
-worker in [vectorizer-worker.md].
+Ensure that the new integration is documented.
+- Document the new database function in [vectorizer-api-reference.md].
+- Document any changes to the vectorizer worker in [vectorizer-worker.md].
+- Add a new row into the [Features per model table in the README file](/README.md#supported-features-per-model) indicating all the supported features.
```
Comment (billy-the-fish) [inline docs/adding-embedding-integration.md:58]:
```suggestion
- Add a new row in [Supported features in each model](/README.md#supported-features-in-each-model) for your worker.
```

**Review by billy-the-fish — APPROVED** [review]
Few small tw thoughts.

**Review by JamesGuthrie — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -37,6 +37,19 @@ the foundation of most AI applications, pgai makes it easier to leverage your da
 
 [![Auto Create and Sync Vector Embeddings in 1 Line of SQL (pgai Vectorizer)](https://github.com/user-attachments/assets/8a71c774-505a-4335-8b34-cdea9dedb558)](https://youtu.be/ZoC2XYol6Zk?si=atI4XPurEifG0pd5)
 
+# Supported features per model
```
[the remark this answers is not in the document]
Comment (smoya) [reply README.md]: Fixed

File: docs/adding-embedding-integration.md [hunk 4]
```diff
@@ -52,8 +52,9 @@ testing we would like.
 
 ## Documentation
 
-Ensure that the new integration is documented. Document the new database
-function in [vectorizer-api-reference.md], and changes to the vectorizer
-worker in [vectorizer-worker.md].
+Ensure that the new integration is documented.
```
[the remark this answers is not in the document]
Comment (smoya) [reply docs/adding-embedding-integration.md]: Fixed

**Pull request merged**